src/domains/transaction/cucumber/validator-resignation.ts

```diff
@@ -4,24 +4,24 @@ import { buildTranslations } from "../../../app/i18n/helpers";
 import { cucumber, MNEMONICS, mockRequest, visitWelcomeScreen } from "../../../utils/e2e-utils";
 import { goToProfile } from "../../profile/e2e/common";
 import { importWallet } from "../../wallet/e2e/common";
-import { goToDelegateResignationPage } from "../e2e/common";
+import { goToValidatorResignationPage } from "../e2e/common";
 
 const translations = buildTranslations();
 const sendButton = Selector("button").withText(translations.COMMON.SEND);
 
-const delegateFormStep = {
+const validatorFormStep = {
 	"Given Alice has navigated to the delegate resignation form for a wallet": async (t: TestController) => {
```

Comment:
Fixed this one too but I know there are  a lot of such usages, and it is easy to miss them without running tests, so that is why I am going to ask you guys to bear with me, I added a specific task to address all of these issues when we have testing setup.

https://app.clickup.com/t/86dvc4u21